### DISCUSSION TITLE
You can support additional router devices jdc ax1800 and jdc ax6600

### DISCUSSION THREAD

**Discussion by Doanduy09** [qosmio/openwrt-ipq#42] [Feature Requests]
Can you please support more router devices jdc ax1800 and jdc ax6600, thanks

**Comment by wuruxu** [comment]
I also use jdc ax6600 too

**Comment by qosmio** [comment]
I mainly add NSS support for existing devices in upstream OpenWrt. The only exception is the MX4300, which isn’t upstream yet; I currently own this device and can personally verify that everything works with each update I push.

Are there branches that support these devices? Minus NSS of course.

**Comment by wuruxu** [comment]
NSS support is very great , these two devices isn't supported in upstream openwrt yet.

**Comment by Doanduy09** [comment]
> Tôi chủ yếu thêm hỗ trợ NSS cho các thiết bị hiện có trong OpenWrt thượng nguồn. Ngoại lệ duy nhất là MX4300, hiện vẫn chưa được thượng nguồn; hiện tại tôi sở hữu thiết bị này và có thể tự mình xác minh rằng mọi thứ đều hoạt động với mỗi bản cập nhật tôi đưa ra.
> 
> Có nhánh nào hỗ trợ các thiết bị này không? Tất nhiên là trừ NSS.

Currently I see this branch supports jdc ax 1800 and jdc ax6600, but when I build the rom to test, I see that it doesn't support wifi very well.
https://github.com/LiBwrt-op/openwrt-6.x
https://github.com/LiBwrt-op/openwrt-6.x/tree/main/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom

**Comment by breeze303** [comment]
You can try the firmware here，Both devices have been using this firmware for close to half a year now, and there are no major problems with it so far
https://github.com/breeze303/openwrt-ci/releases/tag/IPQ60XX-ALL

**Comment by Doanduy09** [comment]
> You can try the firmware here，Both devices have been using this firmware for close to half a year now, and there are no major problems with it so far https://github.com/breeze303/openwrt-ci/releases/tag/IPQ60XX-ALL

thank you very much

**Comment by Doanduy09** [comment]
I built https://github.com/LiBwrt-op/openwrt-6.x compare https://github.com/VIKINGYFY/immortalwrt , same position then see that jdc AX6600 of LiBwrt is slower than VIKINGYFY, I tried many times.. the result is still the same.

**Comment by qosmio** [comment]
@breeze303 I see you've committed a few changes for jdcloud ax1800/ax6600 in your fork, have you opened any PR to have the work committed upstream?

**Comment by breeze303** [comment]
> [@breeze303](https://github.com/breeze303) I see you've committed a few changes for jdcloud ax1800/ax6600 in your fork, have you opened any PR to have the work committed upstream?

Some of my changes may not pass upstream scrutiny, so I haven't opened any PR.
